dotnetv3/README.md

```diff
@@ -129,6 +129,17 @@ If you want more information, run:
 dotnet test -l "console;verbosity=detailed"
 ```
 
+## Docker image (Beta)
+In the near future, this example code will be available in a container image 
+hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). This image will be pre-loaded 
+with all .NET examples with dependencies pre-resolved, allowing you to explore
+these examples in an isolated environment.
+
+⚠️ As of January 2023, the [SDK for .NET v3 image](https://gallery.ecr.aws/aws-docs-sdk-examples/dotnetv3) is available on ECR Public but is still
```

Comment:
Here, and in other READMEs, check whether the â ï¸ characters were intentional